tasks/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-clap = "3.1"
+clap = "4.4.4"
```

Comment:
please hardcode the version here, it's coming from core

Reply:
Fixed